bugbug/repository.py

```diff
@@ -118,16 +120,32 @@ def download_commits(repo_dir):
     commits_num = len(commits)
 
     commits_by_author = {}
+    commits_by_author_90_days = {}
 
     global author_experience
-    for commit in commits:
+    global author_experience_90_days
+
+    for ind, commit in enumerate(commits):
```

Comment:
The double loop is going to be really expensive, please check how long it takes to run this (you can return before the processpoolexecutor).

Reply:
I'll check it out. Is there a way to avoid nested loops here? For every commit, we'd have to iterate up to all commits after it to check the time gap, right?